examples/tsconfig.json

```diff
@@ -2,9 +2,7 @@
   "extends": "@tsconfig/recommended",
   "compilerOptions": {
     "outDir": "dist",
-    "lib": [
-      "ESNext",
-    ],
+    "lib": ["ESNext", "DOM"],
```

Comment:
`fetch` was undefined

Reply:
We do not want all of the DOM library as this is not a browser library. Where was fetch undefined?

Reply:
I thought for the examples that would be ok. 

```
../langchain/document_loaders/cheerio_web_base.ts(40,28): error TS2304: Cannot find name 'fetch'.
examples/src/document_loaders/cheerio_web.ts
```

Reply:
I see. To be honest the examples is something we want to move either into the docs or integration tests, so I'm okay with your change for now